recipes/xsum/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xsum" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21ff547dbb64e8ddefd376f2889349c7a94fce69e7fc417fc661068b298473b1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python 
+    - pip
+    - pybind11
+  run:
+    - python
+    - pybind11
+    - libcxx
+    - numpy
+
+test:
+  imports:
+    - numpy
```

Comment:
You can remove this numpy test

Reply:
@scopatz thanks for catching this. I removed it.